hub/policy/src/main/java/uk/gov/ida/hub/policy/domain/state/CountrySelectedState.java

```diff
@@ -18,7 +17,7 @@
     // TODO: Record matching service entity id
     private String countryEntityId;
     private final Optional<String> relayState;
-    private List<LevelOfAssurance> levelsOfAssurance = Collections.EMPTY_LIST;
+    private List<LevelOfAssurance> levelsOfAssurance;
```

Comment:
Would this ever accidentally cause an NPE?

Reply:
Nope. Because levelsOfAssurance is assigned value in the constructor! Whatever creates the object has to supply a value. If is supplies a null then this default value doesn't prevent a NPE anyway....